view/frontend/web/js/view/payment/method-renderer/pmclain_authorizenetcim.js

```diff
@@ -150,7 +150,7 @@ define(
         var self = this;
         var acceptjsDep = 'acceptjs';
 
-        if (self.getIsTest() == "1") {
+        if (self.getIsTest() === ?) {
```

Comment:
The "?" in the previous comment was only the sentence punctuation. The line should be:  
`if (self.getIsTest() === "1") {`  
Sorry for the confusion.

Reply:
haha, no problem. Seemed odd but thought you knew something I didn't